recipes/scalecodec/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "scalecodec" %}
+{% set repo_name = "py-scale-codec" %}
+{% set version = "1.2.7" %}
+{% set sha256 = "d5723891dba332200017e881ab839c01a7b035863df1585a524c209cfcea5aed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/polkascan/{{ repo_name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --use-pep517 -vvv
+  script_env:
+      - GITHUB_REF=refs/tags/v{{ version }}
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8.0
```

Comment:
This does not match upstream.

https://github.com/polkascan/py-scale-codec/blob/4dbd98a7b66a0de94a18c315d4b678343968884b/setup.py#L172